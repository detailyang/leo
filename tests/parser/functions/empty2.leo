/*
namespace: Parse
expectation: Pass
*/

function x() {}