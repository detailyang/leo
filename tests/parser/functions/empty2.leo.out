---
namespace: Parse
expectation: Pass
outputs:
  - name: ""
    expected_input: []
    imports: []
    circuits: {}
    functions:
      "{\"name\":\"x\",\"span\":\"{\\\"line_start\\\":3,\\\"line_stop\\\":3,\\\"col_start\\\":10,\\\"col_stop\\\":11,\\\"path\\\":\\\"empty2.leo\\\",\\\"content\\\":\\\"function x() {}\\\"}\"}":
        annotations: []
        identifier: "{\"name\":\"x\",\"span\":\"{\\\"line_start\\\":3,\\\"line_stop\\\":3,\\\"col_start\\\":10,\\\"col_stop\\\":11,\\\"path\\\":\\\"empty2.leo\\\",\\\"content\\\":\\\"function x() {}\\\"}\"}"
        input: []
        output: ~
        block:
          statements: []
          span:
            line_start: 3
            line_stop: 3
            col_start: 14
            col_stop: 16
            path: empty2.leo
            content: "function x() {}"
        span:
          line_start: 3
          line_stop: 3
          col_start: 1
          col_stop: 16
          path: empty2.leo
          content: "function x() {}"
